.github/scripts/deploy.sh

```diff
@@ -26,3 +32,8 @@ then
     aws s3api put-object --bucket $bucket --key "content/$patchUpdate/LICENSE-THIRD-PARTY" --body LICENSE-THIRD-PARTY --cache-control max-age=7200
     aws s3api put-object --bucket $bucket --key "content/$patchUpdate/LICENSE" --body LICENSE --cache-control max-age=7200
 fi
+
+echo "1 min bake time"
+sleep 60
+
+echo "Completed deployment"
```

Comment:
issue: I think the intent here is to wait until the backend has fetched the new `versions` file before deploying the version, in which case this should be at line 22.

However, I don't think we should add this sleep for the following reasons:
1. I think `current` and `versions` should only contain versions which are available (for example, if some automation reads the value of `current`, but that version of `cwr.js` is not available, it could break the automation).
2. Sleeping for one minute wastes compute time in the CI/CD pipeline.

In practice, I doubt the delay will have significant impact on monitoring the health of the web client, and if it does, we can re-visit this.

Reply:
Agree, removed.